common/src/main/java/org/apache/uniffle/common/ShuffleDataResult.java

```diff
@@ -79,6 +79,10 @@ public ByteBuffer getDataBuffer() {
     return buffer.nioByteBuffer();
   }
 
+  public int getDataBufferLength() {
```

Comment:
Could we unify `getData().length()` and `getDataBuffer().remaining()`?

Reply:
Done.